tests/lora/test_lora_layers_flux.py

```diff
@@ -535,17 +515,107 @@ def test_lora_expanding_shape_with_normal_lora_raises_error(self):
             "transformer.x_embedder.lora_B.weight": shape_expander_lora_B.weight,
         }
 
-        # We should check for input shapes being incompatible here. But because above mentioned issue is
-        # not a supported use case, and because of the PEFT renaming, we will currently have a shape
-        # mismatch error.
+        # We should check for input shapes being incompatible here.
         self.assertRaisesRegex(
             RuntimeError,
-            "size mismatch for x_embedder.lora_A.adapter-2.weight",
+            "x_embedder.lora_A.weight",
```

Comment:
Why is that part of the comment being removed, and why is `size mismatch` part being removed?

Just to confirm, we still don't support:
- Load N different "normal" loras
- Try to load "shape expander" lora
- Will the N different normal loras be expanded?